tests/hazmat/backends/test_openssl.py

```diff
@@ -609,30 +609,6 @@ def test_password_length_limit(self):
             )
 
 
-class TestGOSTCertificate(object):
```

Comment:
What changed here?

Reply:
We support this string type on all paths and it's not an openssl specific test (so it's now tested in test_x509)